src/client/client.spec.ts

```diff
@@ -1,28 +1,28 @@
 import { Client } from './client';
 import { Rebound } from '../rebound/rebound';
 
-describe("Client:", () => {
+describe('Client:', () => {
   let client: any;
```

Comment:
were you not able to remove this any too ?

Reply:
If I do then the tests will fail because in typescript you should not be able to access them.